mlflow/deployments/databricks/__init__.py

```diff
@@ -335,10 +336,10 @@ def create_endpoint(self, name, config=None, route_optimized=False):
 
         Args:
             name: The name of the serving endpoint to create.
-            config: A dictionary containing the configuration of the serving endpoint to create.
+            config: A dictionary containing either the full API request payload
```

Comment:
We'll need a deprecation indicator for the APIs that we don't want people using anymore so that this notice appears in the API docs